test/integration/minishift.go

```diff
@@ -112,3 +112,20 @@ func (m *Minishift) getOpenShiftUrl() string {
 	cmdOut, _, _ := m.runner.RunCommand("console --url")
 	return strings.TrimRight(cmdOut, "\n")
 }
+
+func (m *Minishift) getRoutingUrlAndVerifyHTTPResponse(partOfResponse, urlRootSuffix, serviceName, nameSpace, assertion, expected string) error {
```

Comment:
This method can then call minishift.getRoute and verifyHTTPResponse.
> If you have a generic getRoute in the Minishift struct, this method can be used in another context as well. If you "merge" the concern of getting the route with the concern of executing the HTTP request and making an assertion, your code becomes much more coupled and less re-usable.

+1. Implemented the changes. Would you please have a look.